### PR TITLE
LRAC-7748 - [C] Clean up Analytics Client before Custom Events release

### PR DESCRIPTION
**Pull request by fcodias**
https://issues.liferay.com/browse/LRAC-7748

cc @kienD

### CODE REVIEW

File: modules/apps/analytics/analytics-client-js/src/analytics.js [hunk 1]
```diff
@@ -12,20 +12,21 @@
...
-
 import uuidv4 from 'uuid/v4';
 
-import Client from './client';
-import EventQueue from './eventQueue';
-import MessageQueue from './messageQueue';
 import middlewares from './middlewares/defaults';
 import defaultPlugins from './plugins/defaults';
+import QueueFlushService from './queueFlushService';
+import EventMessageQueue from './queues/eventMessageQueue';
```
Comment (kienD) [inline modules/apps/analytics/analytics-client-js/src/analytics.js:20]: Alphabetize

File: modules/apps/analytics/analytics-client-js/src/analytics.js [hunk 2]
```diff
@@ -12,20 +12,21 @@
...
+import EventQueue from './queues/eventsQueue';
+import IdentityMessageQueue from './queues/identityMessageQueue';
 import {
 	FLUSH_INTERVAL,
+	QUEUE_PRIORITY_DEFAULT,
 	QUEUE_PRIORITY_IDENTITY,
 	STORAGE_KEY_EVENTS,
 	STORAGE_KEY_IDENTITY,
+	STORAGE_KEY_MESSAGES,
```
Comment (kienD) [inline modules/apps/analytics/analytics-client-js/src/analytics.js:29]: Alphabetize

File: modules/apps/analytics/analytics-client-js/src/analytics.js [hunk 3]
```diff
@@ -512,6 +463,48 @@ class Analytics {
...
+		});
+
+		instance[STORAGE_KEY_MESSAGES] = eventMessageQueue;
+		instance._queueFlushService.addQueue(eventMessageQueue, {
+			priority: QUEUE_PRIORITY_DEFAULT,
+		});
+	}
+
+	/**
+	 * Create member instance of EventMessageQueue to store identity messages.
```
Comment (kienD) [inline modules/apps/analytics/analytics-client-js/src/analytics.js]: Is this comment correct?

File: modules/apps/analytics/analytics-client-js/src/analytics.js [hunk 4]
```diff
@@ -512,6 +463,48 @@ class Analytics {
...
+		});
+
+		instance[STORAGE_KEY_EVENTS] = eventQueue;
+		instance._queueFlushService.addQueue(eventQueue, {
+			priority: QUEUE_PRIORITY_DEFAULT,
+		});
+	}
+
+	/**
+	 * Create member instance of EventMessageQueue to store identity messages.
```
Comment (kienD) [inline modules/apps/analytics/analytics-client-js/src/analytics.js]: Same here.

**Review by kienD — COMMENTED** [review]
Just a couple nitpicks

File: modules/apps/analytics/analytics-client-js/src/analytics.js (hunk 1 above)
Comment (fcodias) [reply modules/apps/analytics/analytics-client-js/src/analytics.js:20]: SF in portal is based on the path :/

**Comment by fcodias** [comment]
updated @kienD! Thanks!!

**Comment by fcodias** [comment]
ci:test:sf

**Comment by fcodias** [comment]
ci:test:relevant

**Comment by fcodias** [comment]
ci:forward

**Comment by fcodias** [comment]
ci:test:relevant

**Comment by fcodias** [comment]
ci:test:relevant

**Comment by fcodias** [comment]
ci:stop

**Comment by fcodias** [comment]
ci:test:relevant

**Comment by fcodias** [comment]
ci:stop

**Comment by fcodias** [comment]
ci:test:relevant

**Comment by fcodias** [comment]
ci:test:relevant

**Comment by fcodias** [comment]
resent https://github.com/fcodias/liferay-portal/pull/35

**Pull request closed without merging**